src/components/screens/sprint-view-screen.tsx

```diff
@@ -86,12 +84,15 @@ const SprintViewScreen = () => {
       setAllocations(fetchedAllocations);
       setTimeEntries(fetchedTimeEntries);
     } catch (error) {
-      console.error("Error fetching data:", error);
+      setError(`${strings.sprintRequestError.fetchError}, ${error}`);
     }
+    setLoading(false);
   };
 
   /**
-   * Calculate project total time spent on the project by the user
+   * Calculate total time allocated to the project for 2 week period
+   * 
+   * @param allocation expected work load of user in minutes
    */
   const totalAllocations = (allocation: Allocations) => {
```

Comment:
This is also repeated in sprint-project-columns and should be exported from a utils file to be used in both places